### ISSUE TITLE
Incompatibility with Bukkit ("Soulbound" enchantment)?

### ISSUE THREAD

**Issue by Nathipha** [Mithion/ArsMagica2#1310]
Version 1.4.0.008

I'm currently playing on a server that uses a few Bukkit plugins. The problem is that the "Soulbound" enchantment isn't working anymore. If you die in any dimension other than the Overworld, the items with the enchantment on it (including e.g. the Earth Armor) don't drop and also aren't stored inside the grave (Extra Utilities), they simply are deleted. The weird thing is: EnderIO also has a "Soulbound" enchantment, which you can only get through their enchanter (you can't get it through the Vanilla enchantment table) and you keep your stuff that is enchanted with their enchantment (unless you burn in the sun as a Witchery vampire, which is a different problem).

Last map we were using: PlayerSQL (https://www.spigotmc.org/resources/playersql.552/) 
This time it's: Essentials (http://dev.bukkit.org/bukkit-plugins/essentials/)

If I set the ID of the AM2 enchantment in the config to 0, will that disable it for the Vanilla enchantment table? This could at least make it impossible to put the enchantment on new equipment and luckily it's possible to disenchant items like the Earth Armor, the Arcane Spellbook,... using MFR's disenchanter. This way you at least won't lose stuff like that, even though it sadly doesn't fix the actual bug.

There are 3 more problems:
1. It doesn't mark the affinities' "buffs" properly in the Occulus. I was already losing hearts from being in the sun (Ender Affinity), even though "Weakness to Sunlight" was still red. Same thing with "Solid Bones" (Earth Affinity) - probably not caused by the incompatibility.
2. This is most likely caused by the incompatibility with the plugin: Your hearts are completely bugging out if you have enough depth in Ender Affinity that you're already affected by the "Weakness to Water" thing. When I was in water, I always lost between 2 and 4 hearts (occassionally even 6) instead of the normal 2 and with the weakness to the sun, I usually got down to 4 hearts (instead of 6).
3. This most likely also happens because of the incompatibility: You sometimes don't get points (only tested with blue), when you level up. I first thought that it only happens if the chunks with the Occulus in it aren't loaded but aren't the points stored in your profile?

**Comment by EoD** [comment]
Duplicate of #1209

**Comment by Nathipha** [comment]
Oh, sorry, I must have missed the other thread.
So, is there no way to fix it atm and can we now only wait and hope that Bukkit plugin developers are going to fix it it eventionally? :/

What's with the other problems, mainly 1. with the buffs not getting marked properly? If I remember correctly, the same thing also happened before before we added Bukkit plugins to the server.

**Comment by LemADEC** [comment]
You easily make a bukkit plugin that'll remove the enchantment on item move events, it'll naturally clear your map.

**Comment by jjtParadox** [comment]
We fixed the AM2 issue with soulbound items, but I also later discovered an issue with Cauldron and it's derivatives that cause soulbound items to disappear. I reported that to them, and they are working on it.

I think this issue should be closed, as we squashed a lot of bugs with this next update and KCauldron has fixed many as well. If those other bugs are still present after the next update, open a new issue and we'll look into it.
